trin-execution/src/storage/evm_db.rs

```diff
@@ -253,24 +258,19 @@ impl EvmDB {
         {
             let plain_state_storage_timer = start_commit_timer("storage:plain_state_storage");
 
-            let timer = start_commit_timer("storage:fetch_account_from_db");
             let address_hash = keccak256(address);
-            let rocks_account: Option<RocksAccount> =
-                self.db.get(address_hash)?.map(|raw_account| {
-                    Decodable::decode(&mut raw_account.as_slice())
-                        .expect("valid account should be decoded")
-                });
-            stop_timer(timer);
 
-            let rocks_account = if wipe_storage && rocks_account.is_some() {
-                let rocks_account = rocks_account.expect("We already checked that it is some");
-                self.delete_account_storage(address_hash, rocks_account, false)?
-            } else {
-                rocks_account
-            };
+            if wipe_storage {
+                self.wipe_account_storage(address_hash, false, "storage:wipe_storage")?;
+            }
 
             if !storage.is_empty() {
-                self.commit_storage_changes(address_hash, rocks_account, storage)?;
+                // review comment: note that commit_storage_changes would have to load RocksAccount
```

Comment:
nit: this was meant for you, not to leave it in the code :), so please delete it